src/components/Table/Table.test.jsx

```diff
@@ -306,18 +305,18 @@ describe('Table', () => {
     expect(renderRowCountLabel2).toHaveLength(0);
   });
 
-  it('enter key should trigger onDownload', () => {
+  it('click should trigger onDownload', () => {
```

Comment:
do we not still want to test the keyboard functionality?

Reply:
we can't @davidicus because the simulate doesn't really simulate button presses, it actually triggers the onKeyDown handler which doesn't exist anymore.